modules/platform/nuxeo-platform-task/nuxeo-platform-task-core/src/main/resources/OSGI-INF/task-listener-contrib.xml

```diff
@@ -8,6 +8,11 @@
 			name="removeProcessForDeletedDocument">
 			<event>aboutToRemove</event>
 		</listener>
+		<listener async="true" postCommit="false"
+			class="org.nuxeo.ecm.platform.task.core.listener.DeleteTaskForDeletedDocumentRouteListener"
+			name="removeTasksForDeletedDocumentRoute">
+			<event>aboutToRemove</event>
```

Comment:
Could you format it correctly with spaces? (I know the rest of the file is using tabs but for added code use spaces 😃 )
You could even correctly format the whole file if you want 😬

Reply:
just discovered eclipse could format xml...sometimes